librz/include/rz_cons.h

```diff
@@ -1070,6 +1070,19 @@ typedef struct rz_line_hist_t {
 	bool do_setup_match;
 } RzLineHistory;
 
+typedef enum {
+	UNDO_DELETE = -1,
```

Comment:
are these values really important? If so, maybe add a doxygen documentation to the whole `enum` and explain a bit more.

Reply:
In some way, I have to tell the undo system whether a single action is to delete text or insert a text, and how many actions does a group of an undo or redo have (especially in vi mode, the number of contained actions will increase). The reason that the enum has a value was just to make it easy to reverse the type of an action.
However, I now am thinking I should do this with another option, which is to distinguish text to be deleted (in the sense of undo) and text to be inserted, and label each entry with an identifer without having special entries at the beginning and end. This might be more straightforward and readable.